db/migrate/20150903081526_add_foreman_tasks_bookmarks.rb

```diff
@@ -0,0 +1,25 @@
+class AddForemanTasksBookmarks < ActiveRecord::Migration
+  def up
+    Bookmark.where(:name => "pending", 
```

Comment:
I would recude the number of different bookmarks: this states often don't tell anything to the end user. I would suggest two bookmars:

* running: state = running
* failed: state = paused OR result = error OR result = warning